backups/object-backup/object-backup.py

```diff
@@ -74,22 +74,32 @@ def get_archive_key(key):
     return os.path.join(config["archive"]["prefix"], key)
 
 
+def dont_backup_key(key):
+    if key.endswith("-thumb"):
+        return True
+
+
 # It is very easy to configure it to upload to the wrong bucket, so this checks that at least 80
 # out a random 100 recordings are already on the target bucket. Meaning it's probably the correct bucket.
+## TODO: Make an API request to the server for getting a random sample of keys from the target bucket.
 print(
     "Check that some files already match as a way of checking that the correct buckets are being/prefix used."
 )
 keys = []
 i = 0
-for obj in local_bucket.objects.page_size(10000):
+keys_sample_size = 10000
+for obj in local_bucket.objects.page_size(1000):
```

Comment:
is reducing 10000 to 1000 intended?

Reply:
Yeah that is fine, just changes how many objects it gets in one query/page.